ChameleonForms/FieldGenerators/Handlers/BooleanHandler.cs

```diff
@@ -25,7 +25,8 @@ public BooleanHandler(IFieldGenerator<TModel, T> fieldGenerator)
         /// <inheritdoc />
         public override bool CanHandle()
         {
-            return GetUnderlyingType(FieldGenerator) == typeof(bool);
+            return GetUnderlyingType(FieldGenerator) == typeof(bool)
+                && !HasEnumerableValues(FieldGenerator);

```

Comment:
Doesn't make sense to support `IEnumerable<bool>`. Didn't bother adding test coverage since it's not something that makes any sense (and this handler would have thrown an exception if anyone tried to use it that way anyway since it casts to bool).